airbyte-config/config-models/src/main/resources/types/ConnectorBuilderProject.yaml

```diff
@@ -21,6 +21,8 @@ properties:
   actorDefinitionId:
     type: string
     format: uuid
+  hasDraft:
```

Comment:
why are we adding a flag instead of checking if `manifestDraft == null`?
is there a risk the two fields could get out of sync?

Reply:
The flag is only in the POJO, not in the database - it's to avoid loading the full draft from database into Java memory just to check whether it's null